notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,261 @@
+     ]
+    }
+   ],
+   "source": [
+    "NOTEBOOK_NAME = 'sar_movielens.ipynb'\n",
+    "EXPERIMENT_NAME = 'sdk-papermill'\n",
+    "exp_directory = 'test_dir' + EXPERIMENT_NAME\n",
+    "# create experiment directory\n",
+    "if not path.exists(exp_directory):\n",
+    "    makedirs(exp_directory)\n",
+    "notebook_path = path.join(exp_directory, NOTEBOOK_NAME)\n",
+    "shutil.copy(NOTEBOOK_NAME, notebook_path)\n",
+    "        \n",
+    "exp = Experiment(workspace=ws, name=EXPERIMENT_NAME)\n",
+    "\n",
+    "run_config_aml = RunConfiguration()\n",
+    "run_config_aml.target = \"amlcompute\" # possible to use different compute targets\n",
+    "run_config_aml.amlcompute.vm_size = 'STANDARD_D2_V2'\n",
+    "run_config_aml.environment.docker.enabled = True\n",
+    "run_config_aml.environment.docker.base_image = DEFAULT_CPU_IMAGE\n",
+    "run_config_aml.environment.python.user_managed_dependencies = False\n",
+    "run_config_aml.auto_prepare_environment = True\n",
+    "run_config_aml.environment.python.conda_dependencies = CondaDependencies(conda_dependencies_file_path='../../myenv2.yaml')\n",
+    "#run_config_aml.environment.python.conda_dependencies = CondaDependencies(conda_dependencies_file_path='../../reco_base.yaml')\n",
+    "\n",
+    "handler = PapermillExecutionHandler(kernel_name='python3')\n",
+    "\n",
+    "cfg = NotebookRunConfig(source_directory='../../',\n",
+    "                                notebook='notebooks/00_quick_start/' + NOTEBOOK_NAME,\n",
+    "                                output_notebook='outputs/out.ipynb',\n",
+    "                                parameters={\"MOVIELENS_DATA_SIZE\": \"100k\", \"TOP_K\": 10},\n",
+    "                                run_config=run_config_aml)\n",
+    "run = exp.submit(cfg)\n",
+    "run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You should see simialar output as below\n",
+    "![Experiment submit output](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/quickstart-get-started/view_exp.png)\n",
+    "Azure Portal looks like this\n",
+    "![Azure Portal Experiment](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/quickstart-get-started/web-results.png)"
```

Comment:
This image does not match what's in the code. It's for my-first-experiment from the notebooks not the sar notebook. It should match what the example is in this notebook.

Reply:
Now using actual screenshot